lvk/vulkan/VulkanClasses.cpp

```diff
@@ -2145,15 +2145,23 @@ void lvk::CommandBuffer::cmdBeginRendering(const lvk::RenderPass& renderPass, co
   ctx_->checkAndUpdateDescriptorSets();
   ctx_->bindDefaultDescriptorSets(wrapper_->cmdBuf_, VK_PIPELINE_BIND_POINT_GRAPHICS);
 
+#ifndef __APPLE__
   vkCmdBeginRendering(wrapper_->cmdBuf_, &renderingInfo);
+#else
+  vkCmdBeginRenderingKHR(wrapper_->cmdBuf_, &renderingInfo);
```

Comment:
Let's assign `vkCmdBeginRenderingKHR` to  `vkCmdBeginRendering` at the initialization to prevent this branching here.

The `VkRenderingInfo` struct should be exactly the same.